icons/aperture-2.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="10" />
+  <circle cx="12" cy="12" r="4" />
+  <path d="M14.8 9.2 8.5 2.8" />
+  <path d="m14.8 14.8 6.4-6.3" />
+  <path d="m9.2 14.8 6.3 6.4" />
+  <path d="m9.2 9.2-6.4 6.3" />
```

Comment:
We would need an extra point of precision here:
![image](https://github.com/lucide-icons/lucide/assets/17746067/214ea11b-9dd2-43cd-a4c3-a31d8cdace34)

On the other hand, I'm not sure about this icon to begin with, I really don't want to add any more `foobar-2` type of icons unless absolutely necessary, and while it's a bit dense, I find the current aperture icon much more readily recognisable.